src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -179,18 +179,22 @@ public HttpResponseMessage GetNiceUrl(int id)
         /// <returns></returns>        
         [FilterAllowedOutgoingContent(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IContent>>), "Items")]
         public PagedResult<ContentItemBasic<ContentPropertyBasic, IContent>> GetChildren(
-            int id, 
-            int pageNumber = 0,  //TODO: This should be '1' as it's not the index
-            int pageSize = 0, 
-            string orderBy = "SortOrder", 
-            Direction orderDirection = Direction.Ascending, 
-            string filter = "")
+                int id,
+                int pageNumber = 0,  //TODO: This should be '1' as it's not the index
+                int pageSize = 0,
+                string orderBy = "SortOrder",
+                Direction orderDirection = Direction.Ascending,
+                int? orderBySystemField = 2,

```

Comment:
@aadfPT I'm trying to figure out why this is an integer param, any ideas? The other controllers have this as a boolean so need to know why/how this is an integer.